regcomp_trie.c

```diff
@@ -517,21 +517,28 @@ is the recommended Unicode-aware way of saying
            : ( state==1 ? special : 0 )                                 \
       )
 
-#define TRIE_BITMAP_SET_FOLDED(trie, uvc, folder)           \
-STMT_START {                                                \
-    TRIE_BITMAP_SET(trie, uvc);                             \
-    /* store the folded codepoint */                        \
-    if ( folder )                                           \
-        TRIE_BITMAP_SET(trie, folder[(U8) uvc ]);           \
-                                                            \
-    if ( !UTF ) {                                           \
-        /* store first byte of utf8 representation of */    \
-        /* variant codepoints */                            \
-        if (! UVCHR_IS_INVARIANT(uvc)) {                    \
-            TRIE_BITMAP_SET(trie, UTF8_TWO_BYTE_HI(uvc));   \
-        }                                                   \
-    }                                                       \
-} STMT_END
+/* Helper function for make_trie(): set a trie bit for both the character
+ * and its folded variant, and for the first byte of a variant codepoint,
+ * if any */
+
+STATIC void
+S_trie_bitmap_set_folded(pTHX_ RExC_state_t *pRExC_state,
+    reg_trie_data *trie, U8 uvc, const U8 * folder)
+{
+    TRIE_BITMAP_SET(trie, uvc);
+    /* store the folded codepoint */
+    if ( folder )
+        TRIE_BITMAP_SET(trie, folder[(U8) uvc ]);
```

Comment:
The `(U8)` cast is now redundant.

Reply:
IMO it shouldn't be called uvc when this is a function. I think the argument should be changed to cp or ch instead. "uvc" means "UV Codepoint", if its not a uv it should have a different name, like uch or something like that.